packages/devtools_app/lib/src/charts/flame_chart.dart

```diff
@@ -88,9 +88,9 @@ abstract class FlameChart<T, V> extends StatefulWidget {
 
   final ValueListenable<V> selectionNotifier;
```

Comment:
selectionNotifier.value could be null. This should have type `ValueListenable<V?>`